vector/v.info/v.info.html

```diff
@@ -136,6 +136,49 @@ <h3>Basic metadata information in shell script style</h3>
 bottom=0.000000
 </pre></div>
 
+<h3>Output in JSON format</h3>
+<div class="code"><pre>
+{
+    "name": "geology",
+    "mapset": "PERMANENT",
+    "project": "nc_spm_08_grass7",
+    "database": "\/grassdata",
+    "title": "North Carolina geology map (polygon map)",
+    "scale": 1,
+    "creator": "helena",
+    "organization": "NC OneMap",
+    "source_date": "Mon Nov  6 15:48:53 2006",
+    "timestamp": null,
+    "format": "native",
+    "level": 2,
+    "num_dblinks": 1,
+    "attribute_layer_number": 1,
+    "attribute_layer_name": "geology",
+    "attribute_database": "\/grassdata\/nc_spm_08_grass7\/PERMANENT\/sqlite\/sqlite.db",
+    "attribute_database_driver": "sqlite",
+    "attribute_table": "geology",
+    "attribute_primary_key": "cat",
+    "projection": "Lambert Conformal Conic",
+    "digitization_threshold": 0,
+    "comment": "",
+    "north": 318117.43741634465,
+    "south": 10875.827232091688,
+    "east": 930172.31282271142,
+    "west": 123971.19498978264,
+    "top": 0,
+    "bottom": 0,
+    "nodes": 2724,
+    "points": 0,
+    "lines": 0,
+    "boundaries": 3649,
+    "centroids": 1832,
+    "areas": 1832,
+    "islands": 907,
+    "primitives": 5481,
+    "map3d": false
+}
+</pre></div>
+
 <h2>PYTHON</h2>
 
```

Comment:
Here you can add an example of using the json output format with other Python data science.

For example

```python
import grass.script as gs
import pandas as pd

# Run v.info command
busstops = gs.run_command("v.info", map="busstopsall", format="json")

# Load data into dataframe
df = pd.DataFrame([busstops])

# Display the DataFrame
print(df)
```

Reply:
Thanks for the example @cwhite911, I couldn't come up with an example of my own as I am unfamiliar with how Grass GIS can be used in data science tools. Hence, I copied your example verbatim for now.